.circleci/config.yml

```diff
@@ -119,3 +148,10 @@ workflows:
             branches:
```

Comment:
Nit: could we rename this workflow to `update-test262-status`? `master` doesn't explain much.